test/multifile/typealias/two-modules/library.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: true
```

Comment:
Can you move this test to test/Serialization/ instead, and put the helper file in the Inputs directory? test/multifile/ is supposed to be for intra-module multi-file issues.